### PR TITLE
Added wrapscan deployment to deployment manifests

### PR DESCRIPTION
**Pull request by nerfZael**

### CODE REVIEW

**Review by dOrgJelli — CHANGES_REQUESTED** [review]
I propose that we first do this for the interface, and then we use the interface's new URI within all wraps, and then deploy those.

**Comment by dOrgJelli** [comment]
Additionally, I realize that if we use the new wrapscan URIs in the schemas, we'll need to then resolve the URI. Here is what I propose we do:
1. Deploy `uri-resolver@1.0.0` to wrapscan (NOTE: we reset version to `1.0.0` for all wraps)
2. Change all `implementations/*` to use `wrap/uri-resolver@1.0`
3. In order to build these, we'll need to modify `import_abis` in `polywrap.yaml`
4. Deploy all `...-uri-resolver@1.0.0` implementations.
5. Finish `wrapscan-uri-resolver@1.0.0`, publish.
6. Update the JS polywrap client to include the `wrapscan-uri-resolver`.
7. Remove `import_abis` from all uri-resolver implementations.

**Comment by dOrgJelli** [comment]
Also some other TODOs I found:
- [ ] Update all URIs to use `wrap/` URIs (not all are possible right away)
- [ ] Update all wraps to use new docs manifest + update READMEs
- [ ] Remove `-ext` and `Extension` where applicable.
- [ ] Make sure all `polywrap.yaml` project names match the wrapscan package name in `polywrap.deploy.yaml`.
- [ ] Reset all uri-resolver versions to `1.0.0`
- [ ] Use `wrap/uri-resolver@1.0` in all implementations (accept patches)

I started working on this here, but then "backed off" to focus on my main tasks: https://github.com/polywrap/uri-resolver-extensions/pull/39

**Comment by nerfZael** [comment]
> Additionally, I realize that if we use the new wrapscan URIs in the schemas, we'll need to then resolve the URI. Here is what I propose we do:
> 
> 1. Deploy `uri-resolver@1.0.0` to wrapscan (NOTE: we reset version to `1.0.0` for all wraps)
> 2. Change all `implementations/*` to use `wrap/uri-resolver@1.0`
> 3. In order to build these, we'll need to modify `import_abis` in `polywrap.yaml`
> 4. Deploy all `...-uri-resolver@1.0.0` implementations.
> 5. Finish `wrapscan-uri-resolver@1.0.0`, publish.
> 6. Update the JS polywrap client to include the `wrapscan-uri-resolver`.
> 7. Remove `import_abis` from all uri-resolver implementations.

We could do 1. and then 6. and after that we do 2, etc
This would, I think, allow us to skip the import_abi stuff

**Comment by nerfZael** [comment]
> Also some other TODOs I found:
> 
> * [ ]  Update all URIs to use `wrap/` URIs (not all are possible right away)
> * [ ]  Update all wraps to use new docs manifest + update READMEs
> * [ ]  Remove `-ext` and `Extension` where applicable.
> * [ ]  Make sure all `polywrap.yaml` project names match the wrapscan package name in `polywrap.deploy.yaml`.
> * [ ]  Reset all uri-resolver versions to `1.0.0`
> * [ ]  Use `wrap/uri-resolver@1.0` in all implementations (accept patches)
> 
> I started working on this here, but then "backed off" to focus on my main tasks: #39

Looks good, just one note.
Can we use `@1` instead of `@1.0`?
This allows resolvers to take advantage of any new feature that comes out in the base interface.

Unrelated, there's no difference between `@1.0.0` and `@1.0` for interfaces, right?

**Comment by dOrgJelli** [comment]
> We could do 1. and then 6. and after that we do 2, etc
> This would, I think, allow us to skip the import_abi stuff

This sounds like the best approach, agreed.

> Can we use @1 instead of @1.0?

No, because if the base interface releases new "features" then the implementations must update themselves to support these new features. Also yes `1.0.0` is equal to `1.0` for interfaces, but we can use patch to update README and comments.

File: implementations/ens/contenthash/package.json [hunk 1]
```diff
@@ -1,20 +1,19 @@
 {
-  "name": "template-wasm-rs",
-  "description": "Polywrap Rust Wrapper Template",
+  "name": "ens-contenthash-uri-resolver-wrap",
+  "description": "ENS contenthash resolver wrap for Polywrap",
   "private": true,
-  "version": "0.9.4",
+  "version": "1.0.0",
```
Comment (dOrgJelli) [inline implementations/ens/contenthash/package.json:5]: Just FYI for future reference, this package.json is only used for downloading dependencies, we don't need a version or description. Name is required though IIRC.

File: implementations/ens/ipfs-contenthash/polywrap.deploy.yaml [hunk 2]
```diff
@@ -7,3 +8,11 @@ jobs:
         uri: fs/./build
         config:
           gatewayUri: https://ipfs.wrappers.io
+      - name: http_deploy
+        package: http
+        uri: $$ipfs_deploy
+        config:
+          postUrl: https://wraps.wrapscan.io/r/wrap/ens-ipfs-contenthash-uri-resolver@1.0.0
```
Comment (dOrgJelli) [inline implementations/ens/ipfs-contenthash/polywrap.deploy.yaml]: I'm confused, shouldn't the URL be `/r/polywrap/...` instead of `/r/wrap/...`?

**Review by dOrgJelli — CHANGES_REQUESTED** [review]
Just one final question regarding the URL we're posting to not having the organization "polywrap" in the URL

File: implementations/ens/ipfs-contenthash/polywrap.deploy.yaml (hunk 2 above)
Comment (nerfZael) [reply implementations/ens/ipfs-contenthash/polywrap.deploy.yaml]: Woops, no idea how this happened

**Review by dOrgJelli — APPROVED** [review]

**Pull request merged**